cmd/adhoc.go

```diff
@@ -72,6 +72,10 @@ func (a *Adhoc) NewServer(userConfig legacy.UserConfig, config legacy.Config) (S
 			TerraformRoot: terraform.Root{},
 			GithubRepo:    github.Repo{},
 		},
+		GithubHostname:   userConfig.GithubHostname,
+		GithubAppID:      userConfig.GithubAppID,
+		GithubAppKeyFile: userConfig.GithubAppKeyFile,
+		GithubAppSlug:    userConfig.GithubAppSlug,
 	}
-	return adhoc.NewServer(cfg)
+	return adhoc.NewServer(cfg, globalCfg, ctxLogger)
```

Comment:
nit: don't pass ctxLogger in 2x, instead just define all these in your large Cfg definition. Ideally we can go back and better break that down, but use that struct for the near term